cocotb/share/lib/fli/FliObjHdl.cpp

```diff
@@ -249,14 +249,42 @@ int FliEnumObjHdl::set_signal_value(const int32_t value,
     }
 
     if (m_is_var) {
-        mti_SetVarValue(get_handle<mtiVariableIdT>(),
-                        static_cast<mtiLongT>(value));
+        switch (action) {
+            case GPI_DEPOSIT:
+                mti_SetVarValue(get_handle<mtiVariableIdT>(),
+                                static_cast<mtiLongT>(value));
+                return 0;
+            case GPI_FORCE:
+                LOG_ERROR("Forcing VHDL variables is not suported by the FLI");
```

Comment:
```suggestion
                LOG_ERROR("Forcing VHDL variables is not supported by the FLI");
```
(and in the rest of the file :) ).